activiti-cloud-query-service/activiti-cloud-services-query/activiti-cloud-services-query-repo/src/main/java/org/activiti/cloud/services/query/app/repository/CustomizedTaskRepository.java

```diff
@@ -28,4 +28,11 @@
                                                 VariableValue<?> value,
                                                 Predicate predicate,
                                                 Pageable pageable);
+
+    Iterable<TaskEntity> findAllOverProcessInstance(Predicate predicate);
```

Comment:
is this naming convention used in other places? findAllOver ?

Reply:
not sure if the `findAllOverProcessInstance` conveys the right meaning.
Maybe `findInProcessInstanceScope` describes it better

Reply:
Thx. I will use the suggested conventions